analyzers/tests/SonarAnalyzer.UnitTest/Rules/SymbolicExecution/SymbolicExecutionRunnerTest.cs

```diff
@@ -327,9 +328,25 @@ public void Method()
 
         [TestMethod]
         public void Analyze_Severity_ExecutesWhenAll() =>
-            Verify(@"string s = null;   // Noncompliant    {{Message for SAll}}
-                                        // Noncompliant@-1 {{Message for SMain}}
-                    s.ToString();       // Noncompliant    {{'s' is null on at least one execution path.}} - rule S2259");
+            VerifyCode<TestSERunnerCS>(@"
+using System.Threading;
+namespace Monitor_Conditions
+{
+    class Program
+    {
+        private object obj = new object();
+        public void Method1(bool condition)
```

Comment:
This scaffolding should not be needed. The rule will trigger for local variable too. Using `Verify(` as before should do the trick - adding the `using` on line 422 will solve the namespace issue.